The-Coding-Sharks-UI/src/components/addTrip.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import axios from "axios";
+
+const AddTrip = () => {
+    const [name, setName] = useState("");
+    const [message, setMessage] = useState("");
+
+const submitTrip = async (e) => {
+    e.preventDefault()
+    try {
+      //I think the 400 error is from the syntax for this axios post request
+        const response = await axios.post(
+            `http://localhost:8080/api/trip`, 
+            name
+    // Comment this out when user auth is added
+      ,{
+          withCredentials: true,
+      }
+
+    );
+    setMessage(response.data.message);
+    }
+
+    catch (error) {
+        console.log(`Name: ${name}`);
+
+        setMessage(error.response?.data?.message || "Failed to add trip.");
+    }
+
+
+
+}
```

Comment:
} catch (error) {
        console.error("Error adding trip:", error);
        setMessage(error.response?.data?.message || "Failed to add trip.");
    }
};